### PR TITLE
Tweak UI styling

### PR DESCRIPTION
**Pull request by davkal**
I find the existing UI hard to read. It's good at showing a lot info at high density, but I find that distracting when going through issues (sometimes even via screen sharing). I propose a lighter touch, less dense and more readable. 

- keep main color theme based on Vivid and Sharp: https://visme.co/blog/website-color-schemes/
- remove lots of custom styles and embrace Bulma panel and table styles
- updates Bulma from 0.75 to 0.9
- remove datatable-specific styling

Before: 
Screenshot 2020-11-06 at 14 53 05

After:
Screenshot 2020-11-06 at 14 52 30

### CODE REVIEW

**Comment by davkal** [comment]
The design is just a straw-person really, I do wonder if others share the issues I'm having. FWIW, I've been using triage party more heavily now, and have gotten used to the existing style.

File: site/base.tmpl [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ block "title" .}} {{end}} :: Triage Party</title>
-    <link rel="stylesheet" href="/third_party/bulma/bulma.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
```
Comment (tstromberg) [inline site/base.tmpl:13]: Can you bring the updated bulma into the /third_party directory?

**Review by tstromberg — CHANGES_REQUESTED** [review]
I like the changes you've proposed, and would like to see this get merged if possible as an improvement over the current iteration.

**Comment by tstromberg** [comment]
Closing until this can be revisited.

**Pull request closed without merging**